### ISSUE TITLE
Greenlet for task 9 dies with exception: unsupported operand type(s) for -: 'NoneType' and 'int'

### ISSUE THREAD

**Issue by NoahMarconi** [rotki/rotki#4597]
## Problem Definition

After update can no longer update balances

## Logs

```{log}
[21/07/2022 13:00:32 EDT] ERROR rotkehlchen.api.rest Greenlet with id 4486557008: Greenlet for task 9 dies with exception: unsupported operand type(s) for -: 'NoneType' and 'int'.
Exception Name: <class 'TypeError'>
Exception Info: unsupported operand type(s) for -: 'NoneType' and 'int'
Traceback:
   File "src/gevent/greenlet.py", line 906, in gevent._gevent_cgreenlet.Greenlet.run
  File "rotkehlchen/api/rest.py", line 274, in _do_query_async
  File "rotkehlchen/api/rest.py", line 645, in _query_blockchain_balances
  File "rotkehlchen/utils/mixins/lockable.py", line 46, in wrapper
  File "rotkehlchen/utils/mixins/cacheable.py", line 97, in wrapper
  File "rotkehlchen/chain/manager.py", line 629, in query_balances
  File "rotkehlchen/utils/mixins/lockable.py", line 46, in wrapper
  File "rotkehlchen/utils/mixins/cacheable.py", line 97, in wrapper
  File "rotkehlchen/chain/manager.py", line 1572, in query_ethereum_balances
  File "rotkehlchen/chain/manager.py", line 1520, in query_ethereum_tokens
  File "rotkehlchen/chain/manager.py", line 1489, in _query_ethereum_tokens
  File "rotkehlchen/chain/ethereum/tokens.py", line 163, in query_tokens_for_addresses
  File "rotkehlchen/chain/ethereum/tokens.py", line 82, in detect_tokens_for_address
  File "rotkehlchen/chain/ethereum/tokens.py", line 213, in _get_tokens_balance_and_price
  File "rotkehlchen/inquirer.py", line 503, in find_usd_price
  File "rotkehlchen/inquirer.py", line 358, in _query_oracle_instances
  File "rotkehlchen/chain/ethereum/oracles/uniswap.py", line 275, in query_current_price
  File "rotkehlchen/chain/ethereum/oracles/uniswap.py", line 241, in get_price
  File "rotkehlchen/chain/ethereum/oracles/uniswap.py", line 433, in get_pool_price
```

### System Description

App Version: 1.25.0
Frontend Version: 
Platform: darwin x64 12.4.0
Electron: 18.2.3

**Comment by nebolax** [comment]
Hey @NoahMarconi thank you for reporting this! It would really helps us to solve the problem if you could share full logs or ethereum address(es) that you are tracking with one of the team members. 
You can do it in our Discord: https://discord.gg/kDzUFc7R (note: we have captcha, so have DMs open to pass it).

**Comment by LefterisJP** [comment]
Fixed by #4604
